README.md

```diff
@@ -1 +1,22 @@
 # hw01
+
+I am **Maria Jose Camacho**, second year student at Harris Public Policy School. Before coming to Chicago, I have worked in different agencies in order to define roles, functions and responsibilities at the regulatory, individual and institutional levels to support in the transition of the **criminal justice system** implemented in 20016 in Mexico. 
+
+
+# Work experience
+
+
+## Federal Attorney General’s Office
```

Comment:
I really like your use of 2nd level headers to organize your different work experiences. It makes it very easy to follow.